### PR TITLE
CSSK-1 Docker container with CS simulator and Kafka was prepared

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

File: version [hunk 1]
```diff
@@ -0,0 +1 @@
+1.0
```
Comment (netpoint-dc) [inline version:1]: Change version to 4.9.3

**Review by netpoint-dc — CHANGES_REQUESTED** [review]

**Pull request merged**